patterns/1-initial/cross-team-retrospectives.md

```diff
@@ -0,0 +1,75 @@
+* As a result trusted committers know each other well and understand constraints, prioritization side effects and team dynamics without ever sharing them with contributors.
+* Also contributors form a well knit group.
+* The contribution process is seen as transient and temporary.
+As a result little is invested in forming a shared team of trusted committers and contributors.
+* There is no clear path from contributor to becoming trusted committer - other than becoming a member of the host team.
+
+## Solution
+
+Bring host team and contributors together:
+
+* As a first step it can help to share a meal together and get to know each other.
+* For collaborations running over several weeks establish a monthly 30 minute retrospective meeting that involves everyone who is needed for a successful contribution.
+* Make sure that action items for each restrospective are being followed up upon, ideally check these action items at the beginning of the next retrospective.
+* Keep the agenda of retrospectives stable and predictable: It's already uncomfortable enough to name and resolve collaboration issues.
+
+Example agenda:
+
+* 5 minute checkin so everyone can test their audio setup, silly questions preferred so people can laugh together, reducing overall stress.
+* 5 minute review for action items from last meeting (each item presented by its owner)
+* 10 minutes to gather strengths and weaknesses of the past collaboration time period. Do this as a combination of writing (sticky notes on a digital white board) and verbally explaining the stickies to make sure introverts get involved as well.
+* 2 minutes to put dots against weaknesses that should be addressed in the next cycle.
+Pick the top 1-2 weaknesses.
+* 10 minutes to gather potential remedy actions to address the picked weaknesses.
+Again use time for writing sticky notes to involve everyone.
+* 2 minutes to put dots against action items (each participant may add 2-3 dots), pick at most top 3 items, assign each item two owners - one trusted committers and one contributor.
+* 5 minutes for checkout so everyone can wind down and leave feedback on the meeting.
+
+Caveat: In particular for cases where people have tried to collaborate for a long time already, the initial meeting may need more than 30 minutes.
+
+## Resulting Context
+
+* Trusted committers understand how to improve communication and contribution processes.
+* Contributors understand how to support trusted committers in improving documentation and processes.
+* Likely both uncover issues that are beyond their direct control but also see ways to address these in the organisation adopting InnerSource.
+* Ideally several learnings can be shared with other InnerSource teams so they avoid running into the same trouble.
+* When done regularly after a handful of retrospectives collaboration improves, issues uncovered reduce, turning the session more and more into a lot of positive feedback.
+As a result motivation on both sides increases.
```

Comment:
Is this last sentence meant to be part of the last bullet, or do you want it to be a new paragraph?

Reply:
It was meant to be part of the last bullet point. I vaguely remember being told that when submitting PRs every sentence should be on it's own line to make review easier - could be that that was custom over in the learning path working group though.